scripts/generate_vesting_spec.js

```diff
@@ -0,0 +1,104 @@
+const { decodeAddress, encodeAddress } = require('@polkadot/util-crypto');
+const { u8aToHex, isHex, hexToU8a } = require('@polkadot/util');
+
+const fs = require('fs');
+const csv = require('csv-parser');
+
+const isValidSubstrateAddress = (address) => {
+  try {
+    encodeAddress(isHex(address) ? hexToU8a(address) : decodeAddress(address));
+
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+async function main() {
+  const inputFilePath = process.argv[2];
+  const blockStartDate = process.argv[3] || '15/09/2021';
+  const blockDuration = 3000;
+  const investors = [];
+  const dates = [
+    '16/10/2021',
+    '16/01/2022',
+    '16/04/2022',
+    '16/07/2022',
+    '16/10/2022',
+    '16/01/2023',
+    '16/04/2023',
+    '16/07/2023',
+    '17/10/2023'
```

Comment:
Let's only support the first 45 which (with 3 dates) are the same the last five we code manually

Reply:
okay